@here/harp-mapview/lib/MapView.ts

```diff
@@ -692,6 +693,38 @@ export interface LookAtParams {
     heading: number;
 }
 
+/**
+ * Optional parameters for [[MapView.fitBounds]].
+ */
+export interface FitBoundsOptions {
+    /**
+     * Margin added to view.
+     *
+     * Range:
+     *  - `0` - no margin,
+     *  - `1` - 100% of calculated distance if added
+     *
+     * @default `0.1` meaning 10% is added to calculated distance, so approximately 10% margin is
```

Comment:
Maybe we should reach out to designers what they need. I can imagine they would expect margin to be in pixels.
Can you please check other renderers (including JSAPI and harp-native) what they have

Reply:
@ninok Any other option implementation adds huuge complexity, so if current `margin` is questionable i would
1) remove `margin` from this patch - it's minor
2) check what others and find optimal solution do in followup patch

Reply:
After research, others usually support this as "padding in pixels".

* https://docs.mapbox.com/mapbox-gl-js/api/#map#fitbounds (google maps api does the same)
* https://leafletjs.com/reference-1.6.0.html#fitbounds-options

Reply:
Okay, please remove margin for now and create a separate ticket,so we can get more input.